R/caching.R

```diff
@@ -114,6 +114,19 @@ get_deps_info <- function(repo_dir) {
   }
 }
 
+
+get_description_deps <- function(repo_dir) {
```

Comment:
move these two functions to `utils.R`

Reply:
functions no longer needed after refator